cclient/iota_client_extended_api.c

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include "iota_client_extended_api.h"
+
+static bool is_unused_address(iota_client_service_t const* const serv,
+                              flex_trit_t const* const addr) {
+  // retcode_t ret = RC_OK;
+  int ret_num = 0;
+  tryte_t trytes_addr[NUM_TRYTES_ADDRESS];
+  find_transactions_req_t* find_tran = find_transactions_req_new();
+  find_transactions_res_t* res = find_transactions_res_new();
+
+  // TODO remove flex_trits-trytes convertion.
```

Comment:
Why ?

Reply:
The main idea is using flex_trits inside cclient, therefore `find_transactions_req_add_address` should take filex_trits as the parameter.